test/parallel/test-sqlite-backup.mjs

```diff
@@ -0,0 +1,229 @@
+import * as common from '../common/index.mjs';
+import tmpdir from '../common/tmpdir.js';
+import { join } from 'path';
+import { DatabaseSync } from 'node:sqlite';
+import { describe, test } from 'node:test';
+import { writeFileSync } from 'fs';
+
+let cnt = 0;
+
+tmpdir.refresh();
+
+function nextDb() {
+  return join(tmpdir.path, `database-${cnt++}.db`);
+}
+
+function makeSourceDb() {
+  const database = new DatabaseSync(':memory:');
+
+  database.exec(`
+    CREATE TABLE data(
+      key INTEGER PRIMARY KEY,
+      value TEXT
+    ) STRICT
+  `);
+
+  const insert = database.prepare('INSERT INTO data (key, value) VALUES (?, ?)');
+
+  for (let i = 1; i <= 2; i++) {
+    insert.run(i, `value-${i}`);
+  }
+
+  return database;
+}
+
+describe('DatabaseSync.prototype.backup()', () => {
+  test('throws if path is not a string', async (t) => {
+    const database = makeSourceDb();
+
+    t.assert.rejects(async () => {
+      await database.backup();
+    }, common.expectsError({
+      code: 'ERR_INVALID_ARG_TYPE',
+      message: 'The "destination" argument must be a string.'
+    }));
+
+    t.assert.rejects(async () => {
+      await database.backup({});
+    }, common.expectsError({
+      code: 'ERR_INVALID_ARG_TYPE',
+      message: 'The "destination" argument must be a string.'
+    }));
+  });
+
+  test('throws if options is not an object', async (t) => {
+    const database = makeSourceDb();
+
+    t.assert.rejects(async () => {
+      await database.backup('hello.db', 'invalid');
+    }, common.expectsError({
+      code: 'ERR_INVALID_ARG_TYPE',
+      message: 'The "options" argument must be an object.'
+    }));
+
+    t.assert.rejects(async () => {
+      await database.backup({});
+    }, common.expectsError({
+      code: 'ERR_INVALID_ARG_TYPE',
+      message: 'The "destination" argument must be a string.'
+    }));
+  });
```

Comment:
This assertion doesn't seem to match up with the test goal (throw if options is not an object). I think this can be removed.